android/app/src/main/AndroidManifest.xml

```diff
@@ -31,22 +31,26 @@
             android:name="flutterEmbedding"
             android:value="2" />
     </application>
+    <!-- Required to query activities that can process text, see:
+         https://developer.android.com/training/package-visibility and
+         https://developer.android.com/reference/android/content/Intent#ACTION_PROCESS_TEXT.
 
+         In particular, this is used by the Flutter engine in io.flutter.plugin.text.ProcessTextPlugin. -->
     <queries>
-        <!-- Required to query activities that can process text, see:
-            https://developer.android.com/training/package-visibility and
-            https://developer.android.com/reference/android/content/Intent#ACTION_PROCESS_TEXT.
-
-            In particular, this is used by the Flutter engine in io.flutter.plugin.text.ProcessTextPlugin. -->
-        <!-- <intent>
+        <intent>
             <action android:name="android.intent.action.PROCESS_TEXT"/>
             <data android:mimeType="text/plain"/>
-        </intent> -->
-
+        </intent>
+        <intent>
+            <action android:name="android.intent.action.PROCESS_TEXT"/>
+            <data android:mimeType="text/plain"/>
+        </intent>
         <!-- Launching URLs -->
         <intent>
             <action android:name="android.intent.action.VIEW" />
             <data android:scheme="https" />
         </intent>
     </queries>
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.CAMERA" />
```

Comment:
May need to look into this further. 
It'd be ideal if this app can work without camera, and only accesses camera when needed. 
https://developer.android.com/media/camera/camera-deprecated/camera-api